theme_paptic/views/images.xml

```diff
@@ -59,38 +59,6 @@
     <field name="url">/theme_paptic/static/src/img/pictures/s_carousel_3.jpg</field>
 </record>
 
-<!-- References -->
-<record id="s_reference_demo_image_1" model="theme.ir.attachment">
-    <field name="key">website.s_reference_demo_image_1</field>
-    <field name="name">website.s_reference_demo_image_1</field>
-    <field name="url">/theme_paptic/static/src/img/pictures/s_references_1.png</field>
-</record>
-<record id="s_reference_demo_image_2" model="theme.ir.attachment">
-    <field name="key">website.s_reference_demo_image_2</field>
-    <field name="name">website.s_reference_demo_image_2</field>
-    <field name="url">/theme_paptic/static/src/img/pictures/s_references_2.png</field>
-</record>
-<record id="s_reference_demo_image_3" model="theme.ir.attachment">
-    <field name="key">website.s_reference_demo_image_3</field>
-    <field name="name">website.s_reference_demo_image_3</field>
-    <field name="url">/theme_paptic/static/src/img/pictures/s_references_3.png</field>
-</record>
-<record id="s_reference_demo_image_4" model="theme.ir.attachment">
-    <field name="key">website.s_reference_demo_image_4</field>
-    <field name="name">website.s_reference_demo_image_4</field>
-    <field name="url">/theme_paptic/static/src/img/pictures/s_references_4.png</field>
-</record>
-<record id="s_reference_demo_image_5" model="theme.ir.attachment">
-    <field name="key">website.s_reference_demo_image_5</field>
-    <field name="name">website.s_reference_demo_image_5</field>
-    <field name="url">/theme_paptic/static/src/img/pictures/s_references_5.png</field>
-</record>
-<record id="s_reference_default_image_6" model="theme.ir.attachment">
-    <field name="key">website.s_reference_default_image_6</field>
-    <field name="name">website.s_reference_default_image_6</field>
-    <field name="url">/theme_paptic/static/src/img/pictures/s_references_6.png</field>
-</record>
```

Comment:
Just a question: I couldn't test because the theme screen is currently bugged, but I saw that these images are used in the svg version of this block in `theme_paptic.svg` (they use the `s_reference_demo_image_x` ids)
We agree that it is not an issue, as they will fallback on the default ones (defined in website) ? :slightly_smiling_face: 
Also, because I don't know this, is this the file where the themes previews are defined ? :slightly_smiling_face:

Reply:
Dumb question in fact, other themes do the same so it is OK :+1: